<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskSettings</class>
 <widget class="QDialog" name="TaskSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Folder to backup</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditFrom">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="fromPushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Location of backup</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEditTo"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="toPushButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="uploadCheckBox">
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="shutdownChechBox">
     <property name="text">
      <string>Shutdown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoBackupCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Scheduled backup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>День недели</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Monday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tuesday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wednesday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thursday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Friday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saturday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sunday</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumTime">
      <time>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
